src/main/kotlin/fi/metatavu/lipsanen/projects/milestones/tasks/TaskController.kt

```diff
@@ -156,7 +168,7 @@ class TaskController {
      * @return updated task
      * @throws TaskOutsideMilestoneException if the cascade update goes out of the milestone boundaries
      */
-    suspend fun update(
+    suspend fun applyTaskProposal(
```

Comment:
This is supposed to update a task from REST entity according to the docs. Should it be ```update``` instead of ```applyTaskProposal```?

Reply:
you're right, somehow both methods names updated to the same one. It should be "update", fxied